website/client/src/components/payments/selectUserModal.vue

```diff
@@ -104,16 +97,17 @@
             </div>
           </button>
           <div
-          v-if="currentEvent && currentEvent.promo ==='g1g1'"
-          class="g1g1-cancel d-flex justify-content-center"
-          v-html="$t('cancel')"
-          @click="close()"
+            v-if="currentEvent && currentEvent.promo ==='g1g1'"
+            class="g1g1-cancel d-flex justify-content-center"
+            @click="close()"
+            v-html="$t('cancel')"
           >
-          {{ $t('cancel') }}
-        </div>
-        <div
-        v-else>
-        </div>
+            {{ $t('cancel') }}
+          </div>
+          <div
+            v-else
```

Comment:
An empty div on `v-else`? Is this helpful?

Reply:
seems I left that div from the previous file version 😬  - will be removed